pkg/plugin/twinmaker/handler.go

```diff
@@ -604,3 +605,32 @@ func (s *twinMakerHandler) GetSessionToken(ctx context.Context, duration time.Du
 
 	return info, err
 }
+
+func (s *twinMakerHandler) GetWriteSessionToken(ctx context.Context, duration time.Duration, workspaceId string) (models.TokenInfo, error) {
```

Comment:
Can we avoid the code duplication? Only one line difference, `GetSessionToken` vs `GetWriteSessionToken`